Library/ViewModels/SurveyResponseViewModel.swift

```diff
@@ -164,6 +164,11 @@ private func isSurvey(request: URLRequest) -> Bool {
   return true
 }
 
-private func isStripeElement(_ request: URLRequest) -> Bool {
-  return request.url?.host == "js.stripe.com"
+// Returns true if the url host is of the form *.stripe.com, *.stripe.network, or *.stripecdn.com.
+private func isStripeRequest(_ request: URLRequest) -> Bool {
+  let stripeDomains = ["stripe.com", "stripe.network", "stripecdn.com"]
+
+  guard let host = request.url?.host?.lowercased() else { return false }
+  let withoutSubdomain = host.split(separator: ".").suffix(2).joined(separator: ".")
```

Comment:
Not necessary for this but it might be useful to make a category on URL/String that returns a "base domain" if this kind of pattern is used anywhere else

Reply:
I don't think it is (the only other hosts we care about are kickstarter ones), but agreed - if I see this elsewhere or find myself needing it in the future, I'll pull this out into a utility function then!